examples/mdns_server.rs

```diff
@@ -47,13 +45,14 @@ async fn main() -> Result<(), Error> {
         );
 
     let matches = app.clone().get_matches();
+    let server = matches.value_of("server").unwrap();
```

Comment:
any difference moving line 55 to here?

Reply:
None at all. Must have made the change while testing.